src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -268,15 +269,64 @@
      */
     @Argument(fullName="output_model",
             shortName = "outputModel",
-            doc="If specified, the variant recalibrator will output the VQSR model fit to the file specified by -modelFile or to stdout",
+            doc="If specified, the variant recalibrator will output the VQSR model to this file path.",
             optional=true)
-    private boolean outputModel = false;
+    private String outputModel = null;
 
-    @Argument(fullName="model_file",
-            shortName = "modelFile",
-            doc="A GATKReport containing the positive and negative model fits",
+    /**
+     *  The filename for a VQSR model fit to use to recalibrate the input variants. This model should be generated using
+     *  a previous VariantRecalibration run with the --output_model argument.
+     */
+    @Argument(fullName="input_model",
+            shortName = "inputModel",
+            doc="If specified, the variant recalibrator will read the VQSR model from this file path.",
             optional=true)
-    private String modelReport = null;
+    private String inputModel = null;
+
+    /**
+     * This argument is intended to be used in a more complicated VQSR scheme meant for very large WGS callsets that
+     * require a prohibitive amount of memory for classic VQSR. Given that training data is downsampled once it exceeds
+     * --maxNumTrainingData, reading in additional data to build the model only serves to consume resources. However,
+     * with this argument the output recal file will also be downsampled. The recommended VQSR procedure when using this
+     * argument is to run VariantRecalibrator once with sampling and designate an --output_model file. Then
+     * VariantRecalibrator can be run a second time scattered using that file as an --input_model.  The gather of the recal
```

Comment:
That was the point where I realized "Oh crap I can't just concatenate the tranches."  :)